src/components/Banner/Banner.tsx

```diff
@@ -88,47 +88,48 @@ export const Banner = (props: IBannerProps): React.ReactElement => {
 	}
 
 	return (
-		<ReactTransitionGroup
-			transitionName={cx('&')}
-			transitionEnterTimeout={300}
-			transitionLeaveTimeout={300}
-		>
-			{!isClosed ? (
-				<section
-					{...omitProps(passThroughs, undefined, _.keys(Banner.propTypes))}
-					className={cx(
-						'&',
-						{
-							'&-has-icon': displayedIcon,
-							'&-has-close': isCloseable,
-							'&-primary': kind === 'primary',
-							'&-success': kind === 'success',
-							'&-warning': kind === 'warning',
-							'&-danger': kind === 'danger',
-							'&-info': kind === 'info',
-							'&-small': isSmall,
-							'&-filled': isFilled,
-						},
-						className
-					)}
-				>
-					{displayedIcon ? (
-						<span className={cx('&-icon')}>{displayedIcon}</span>
-					) : null}
-
-					<span className={cx('&-content')}>{children}</span>
-
-					{isCloseable ? (
-						<CloseIcon
-							isClickable
-							size={8}
-							className={cx('&-close')}
-							onClick={handleClose}
-						/>
-					) : null}
-				</section>
-			) : null}
-		</ReactTransitionGroup>
+		<TransitionGroup>
+			<CSSTransition
+				classNames={cx('&')}
+				timeout={300}
+			>
+				{!isClosed ? (
+					<section
+						{...omitProps(passThroughs, undefined, _.keys(Banner.propTypes))}
+						className={cx(
+							'&',
+							{
+								'&-has-icon': displayedIcon,
+								'&-has-close': isCloseable,
+								'&-primary': kind === 'primary',
+								'&-success': kind === 'success',
+								'&-warning': kind === 'warning',
+								'&-danger': kind === 'danger',
+								'&-info': kind === 'info',
+								'&-small': isSmall,
+								'&-filled': isFilled,
+							},
+							className
+						)}
+					>
+						{displayedIcon ? (
+							<span className={cx('&-icon')}>{displayedIcon}</span>
+						) : null}
+
+						<span className={cx('&-content')}>{children}</span>
+
+						{isCloseable ? (
+							<CloseIcon
+								isClickable
+								size={8}
+								className={cx('&-close')}
+								onClick={handleClose}
+							/>
+						) : null}
+					</section>
+				) : <React.Fragment></React.Fragment>}
```

Comment:
This new component will not accept null so we need to send something, in this case I chose an empty fragment for each case.